notifications.yaml

```diff
@@ -727,6 +727,12 @@ blueprint:
               value: low
             - label: "Low (Makes no sound, doesn't appear in status bar)"
               value: min
+    android_high_priority:
+      name: "❕ High Priority Mode"
+      description: "Workaround to ensure notifications are delivered in timely fashion. This will override the Notification Channel Importance with `importance: high`, and set `ttl: 0`.  \n\n`🤖 Android Only`"
```

Comment:
Oh man, seems like I forgot to update the description after testing and porting the changes.